tests/test_inference.py

```diff
@@ -75,14 +69,6 @@ def match_ancestors_ancestors_unfinalised(self, path=None):
         if path is not None:
             ancestor_data.close()
 
-    def test_make_ancestor_data(self):
-        self.make_ancestor_data_unfinalised()
-
-    def test_make_ancestor_data_file(self):
-        with tempfile.TemporaryDirectory(prefix="tsinf_inference_test") as tempdir:
-            filename = os.path.join(tempdir, "samples.tmp")
-            self.make_ancestor_data_unfinalised(filename)
-
     def test_match_ancestors_ancestors(self):
         self.match_ancestors_ancestors_unfinalised()
```

Comment:
Do we test `match_ancestors_ancestors_unfinalised` with the `path` argument anywhere?

Reply:
Yes, on line 78.